source/_docs/blueprint/selectors.markdown

```diff
@@ -43,6 +44,23 @@ This selector does not have any other options; therefore, it only has its key.
 action:
 ```
 
+## Add-on selector
+
+This can only be used on an installation with a Supervisor. For installations
+that do not have that, an error will be displayed.
+
+The add-on selector allows the user to input a add-on slug.
+On the user interface, It will list all installed add-ons and use the slug of the
+selected add-on.
```

Comment:
```suggestion
The add-on selector allows the user to input an add-on slug.
On the user interface, it will list all installed add-ons and use the slug of the
selected add-on.
```